accounts-db/src/ancient_append_vecs.rs

```diff
@@ -548,9 +548,6 @@ impl AccountsDb {
         self.thread_pool_clean.install(|| {
             packer.par_iter().for_each(|(target_slot, pack)| {
                 let mut write_ancient_accounts_local = WriteAncientAccounts::default();
-                self.shrink_ancient_stats
-                    .bytes_ancient_created
-                    .fetch_add(pack.bytes, Ordering::Relaxed);
                 self.write_one_packed_storage(
```

Comment:
The remaining update is in `write_one_packed_storage`:
https://github.com/jeffwashington/solana/blob/93eca3017e62fbeb95cc14ecdfb307a48b9b6c2e/accounts-db/src/ancient_append_vecs.rs#L784-L786